internal/server/placepage/place_page.go

```diff
@@ -275,8 +279,8 @@ func fetchBtData(
 	for place, data := range mergedPlacePageData {
 		finalData := &pb.StatVarSeries{Data: map[string]*pb.Series{}}
 		categoryData[place] = &pb.Categories{}
-		categoryData[place].Category = append(categoryData[place].Category,
-			data.Categories...)
+		categoryData[place].Category = util.MergeDedupe(
+			categoryData[place].Category, data.Categories)
```

Comment:
are these still needed, or can just use data.Categories?

Reply:
Done. Added a sort call to ensure that the output is sorted.